tests/test_job.py

```diff
@@ -1249,3 +1249,22 @@ def test_zero_custom_error_message(hq_env: HqEnv):
         == "Error: Task created an error file, but it is empty"
     )
     # print(table)
+
+
+# def test_resubmit_lost_worker(hq_env: HqEnv):
```

Comment:
While it's nice to show a warning during resubmit, our main use-case here is to just stop re-running the task if it crashes worker too often. So if a task crashed a worker e.g. for 5 times, we should move it to the canceled/failed category, so that it's not started again.

It might be tricky to create a task that crashes workers in tests. Maybe we could create a bash script that is passed the PID of the worker that will execute the task (we know that beforehand in tests) and then the script could `kill` the PID to crash the worker.